### ISSUE TITLE
Looking for a maintainer

### ISSUE THREAD

**Issue by rubenv** [rubenv/angular-gettext#365]
Development of angular-gettext has up till now always been funded by companies I work with, for which I'm very grateful. Sadly the last of those projects will be coming to an end in a couple of months.

With no further sponsoring and a need to feed both my family (with a new kid on the way), it's becoming rather hard for me to find the time to maintain a project that I no longer use myself.

Not that angular-gettext needs much maintenance: as far as I can tell it's feature-complete. Angular (1.x) itself is also frozen, so no large changes will come in Angular are expected. But one never knows what might pop up: unforeseen browser breakage might necessitate changes and fixes, which have to be done with proper QA.

For that reason, if you are an active user of angular-gettext: **we need a new maintainer**. Give me a sign (here or via e-mail) if you are interested.

Some ground rules though:

* Guaranteed backwards compatibility and no breakage *ever*. Unless for a major and critical security problem, all existing projects should keep on working.

* No new features, unless there's an extremely good reason for it (something that's absolutely impossible right now). But no nice-to-haves: simplicity is key.

I'll still keep an eye on things, but I can't spend the required time on this project anymore. On the other hand, I don't want anyone that depends on angular-gettext to be blocked by my lack of time, so I'm happy to relinquish control.

**Comment by calve** [comment]
For the sake of it, please try to not reproduce the `event-stream` failure https://github.com/dominictarr/event-stream/issues/116. The gist of the issue is that the original maintainer gave write access to a malicious actor.

Good luck otherwise and thanks for all the fish :dolphin:

**Comment by rubenv** [comment]
Yeah, totally. We'll need to find a way to do this safely. For one thing, I don't plan to fall of the planet of the earth, so I'll definitely keep an eye on releases (at the very least).
